database/factories/ClinicFactory.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Collection;
+use Lightit\Clinics\Domain\Models\Clinic;
+use Lightit\Doctors\Domain\Models\Doctor;
+
+/**
+ * @extends Factory<Clinic>
+ */
+class ClinicFactory extends Factory
+{
+    protected $model = Clinic::class;
+
+    public function definition(): array
+    {
+        return [
+            'name' => fake()->company() . ' Medical',
+            'address' => fake()->streetAddress() . ', ' . fake()->city()
```

Comment:
```suggestion
            'address' => fake()->address()
```
Why not just `address()`?